### PR TITLE
koops: Check if returned normalized list isnt empty

### PR DESCRIPTION
**Pull request by mkutlak**
Signed-off-by: Martin Kutlak <mkutlak@redhat.com>

### CODE REVIEW

**Comment by ernestask** [comment]
https://github.com/abrt/faf/blob/781b4ccfb8589f0c2e8ebf9c880c308c76e87fba/src/pyfaf/problemtypes/kerneloops.py#L211

The above branch in particular may cause `None` to be indexed.

File: src/pyfaf/problemtypes/kerneloops.py [hunk 1]
```diff
@@ -770,4 +770,7 @@ def check_btpath_match(self, ureport, parser):
 
     def find_crash_function(self, db_backtrace):
         satyr_koops = self._db_backtrace_to_satyr(db_backtrace)
-        return satyr_koops.frames[0].function_name
+        if satyr_koops.frames
```
Comment (ernestask) [inline src/pyfaf/problemtypes/kerneloops.py]: /me facepalms.

Comment (ernestask) [reply src/pyfaf/problemtypes/kerneloops.py]: Guess I should bring my magnifying glass next time.

Comment (mkutlak) [reply src/pyfaf/problemtypes/kerneloops.py]: #metoo

**Pull request merged**